### PR TITLE
[improvement](test)Use show_nested_index_file to check inverted index

### PR DESCRIPTION
**Pull request by qidaye**
## Proposed changes

Use show_nested_index_file to check inverted index instead of calc_file_crc

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

**Comment by qidaye** [comment]
run buildall

**Review by airborne12 — DISMISSED** [review]
LGTM

File: regression-test/suites/inverted_index_p0/index_format_v2/test_create_table_with_format_v2.groovy [hunk 1]
```diff
@@ -20,8 +20,40 @@ import org.codehaus.groovy.runtime.IOGroovyMethods
 suite("test_create_table_with_format_v2", "inverted_index_format_v2"){
     def tableName = "test_create_table_with_format_v2"
 
-    def calc_file_crc_on_tablet = { ip, port, tablet ->
-        return curl("GET", String.format("http://%s:%s/api/calc_crc?tablet_id=%s", ip, port, tablet))
+    def check_nested_index_file = { ip, port, tablet_id, expected_rowsets_count, expected_indices_count, format ->
```
Comment (xiaokang) [inline regression-test/suites/inverted_index_p0/index_format_v2/test_create_table_with_format_v2.groovy:23]: duplicate

**Comment by qidaye** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by csun5285 — APPROVED** [review]
LGTM

**Pull request merged**